cdk/kittyhawk/test/autoscaler.test.ts

```diff
@@ -7,22 +7,37 @@ export function buildAutoscalingChart(scope: Construct) {
 
   /** Autoscaling test **/
   new Application(scope, 'serve', {
-    image: 'pennlabs/website',
-    autoScalingProps: { cpu: 80, memory: 80, requests: 80 },
+    deployment: {
+      image: 'pennlabs/website',
+      autoScalingProps: { cpu: 80, memory: 80, requests: 80 }, // TODO: fix this
```

Comment:
What is `autoScalingProps` and `cpu` supposed to do? I don't see this implemented anywhere in the code

Reply:
We can just remove all the autoscaling stuff from the initial version of kittyhawk IMO. It'll be nice to have in the future bu it's not needed for our MVP.